cmd/beacon-chain/sync/checkpoint/options.go

```diff
@@ -47,22 +47,22 @@ func BeaconNodeOptions(c *cli.Context) ([]node.Option, error) {
 		return []node.Option{opt}, nil
 	}
 
-	if blockPath == "" && statePath == "" {
+	switch {
+	case blockPath == "" && statePath == "":
 		return nil, nil
-	}
-	if blockPath != "" && statePath == "" {
+	case blockPath == "" || statePath == "":
```

Comment:
wouldn't it be better to have separate cases for this?

case blockPath == "":
	return nil, errors.New("--checkpoint-state specified, but not --checkpoint-block. both are required")
case statePath == "":
	return nil, errors.New("--checkpoint-block specified, but not --checkpoint-state. both are required")